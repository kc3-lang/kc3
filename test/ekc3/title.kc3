title = "EKC3 test title"
title_h1 = "<title>\"Title escaped &amp; in title'</title>"
title_h2 = "<h2>Title H2 (raw)</h2>"
puts(EKC3.render_file(__DIR__ + "title.html.ekc3"))
