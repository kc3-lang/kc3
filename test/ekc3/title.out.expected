<!doctype html>
<html>
  <head>
    <title>EKC3 test title</title>
  </head>
  <body>
    <h1>EKC3 test title H1 (silent block)</h1>
    <h1>&lt;title&gt;&quot;Title escaped &amp;amp; in title&#39;&lt;/title&gt;</h1>
    <h2>Title H2 (raw)</h2>
  </body>
</html>

